ChangeLog.md

```diff
@@ -3,7 +3,8 @@
   * _*Master Release*_ - Jun 1, 2022
   * Fixes an issue with previously set randomly generated id values being skipped over in favor of newer values. Now old values, if they exist, are sent up without.    
 
-- v5.1.4
+- ⚠️ v5.1.4  
+Warning: This release has a major configuration issue that incorrectly marks opens as reinstalls. Please use 5.1.5.
```

Comment:
I would remove major, i think the following is better. 

Warning: This release has a configuration issue that incorrectly marks opens as reinstalls. Please use 5.1.5.

Reply:
Done